Resources/views/CRUD/_email_link.html.twig

```diff
@@ -0,0 +1,27 @@
+
+{%- if value is empty -%}
+    &nbsp;
+{%- elseif field_description.options.as_string is defined and field_description.options.as_string -%}
+    {{ value }}
+{%- else -%}
+    {%- set url = '' -%}
+    {%- if field_description.options.subject is defined and field_description.options.subject is not empty -%}
+        {%- set subject = field_description.options.subject -%}
+        {%- set url = 'subject=' ~ subject -%}
+    {%- endif -%}
```

Comment:
It think you could simplify all this greatly by using [default](http://twig.sensiolabs.org/doc/filters/default.html):

Maybe try something like `{% set url = field_description.options.subject|default('') %}`